### ISSUE TITLE
NMM update 0.65.7 Virus?

### ISSUE THREAD

**Issue by RsAddy** [Nexus-Mods/Nexus-Mod-Manager#463]
Was notified of a NMM update and went to the github site to download. The download failed and windows defender notified me of a virus threat. Is this update verified as clean?

**To Reproduce**
Click the newest version link exe on github website for NMM

**Comment by squid-box** [comment]
VirusTotal.com detects `Trojan.Win32.Inject!O`, so it does seem like it :(

**Comment by shinji257** [comment]
Norton flagged Heur.AdvML.C but that is a behavior flag and doesn't necessarily indicate an infection.  The virustotal.com scan was only 1/63.  I've never even heard of CMC Antivirus until now...

https://www.virustotal.com/#/file/c9e7054f87e18e8adcb4965a3a39e5d873da7e199c19e6ad6b6f6bf63de6d886/detection

**Comment by squid-box** [comment]
Good point, I totally misread that results page. 1/63 seems more likely to be a false positive than an actual virus. Windows Defender doesn't find anything either.

**Comment by dyslexicanaboko** [comment]
Game mod managers generally exhibit the behavior of viruses. If you have ever used any game trainers in the past, this is very common. Happened all the time with Diablo 1 and Diablo 2 game trainers among others. There might be a way to clear what the NMM is doing so red flags aren't thrown up. Good one for the backlog I'd say.

**Comment by shinji257** [comment]
I'm going to add that the flags are on the installer.  Once installed the actual application is never flagged.
